tests/testthat/test-utilities-units.R

```diff
@@ -9,10 +9,14 @@ test_that("It can convert from a value in base unit to a target unit", {
   expect_equal(toUnit(par, 1, "l"), 1)
 })
 
-test_that("It can convert from a value in base unit to display unit", {
-  expect_equal(toDisplayUnit(par, 1), 1)
+test_that("It can convert from a value in base unit to a target unit using the dimension name", {
+  expect_equal(toUnit(par$dimension, 1, "ml"), 1000)
 })
 
+test_that("It can convert from a value in mass to a value in mol using a molweight parameter", {

```

Comment:
@pchelle Example
Note that MW internally is in kg/umol. You should never have to worry about it as long as you retrieve the value from the model. It wil lbe in the good unit